add codebyte examples for the Date.getMinutes(), Date.getMonth(), and… (#2322)
Codecademy/docs: content/javascript/concepts/dates/terms/getMonth/getMonth.md

// Output: Time flies, it is now June.
```

## Codebyte Example

> The example below defines a new `Date` object `dateTimeNow`, an array `months` containing the names of the calendar months, uses the `getMonth()` method to extract and store the current month in a new variable `month`, and then logs the current date\time, the current month, and the months remaining in the current year.

Reviewer: ```suggestion
The example below defines a new `Date` object `dateTimeNow`, an array `months` containing the names of the calendar months, and stores the result of the  `getMonth()` method in a new variable `month`. Finally, the current date\time, the current month, and the months remaining in the current year are logged to the console.
```

```codebyte/javascript
let dateTimeNow = new Date();
let months = 
   ['January', 'February', 'March', 'April', 
    'May', 'June', 'July', 'August', 
    'September', 'October', 'November', 'December'];
let month = dateTimeNow.getMonth(); // Uses 0-based indexing; January = 0

console.log(`Date\\Time Now: ${dateTimeNow}\n`);

> console.log(`Current month: ${months[month + 1]}`);

Reviewer: ```suggestion
console.log(`Current month: ${months[month]}`);
```

> console.log(`Months remaining in the current year: ${12 - month}`);

Reviewer: ```suggestion
console.log(`Months remaining in the current year: ${11 - month}`);
```
